schema/query/fuzz.go

```diff
@@ -0,0 +1,51 @@
+// +build gofuzz
+
+package query
```

Comment:
Yeah, I must have missed the filename during review. Sorry about that! I have no experience with go-fuzz.